src/gbz80.7

```diff
@@ -0,0 +1,354 @@
+.Dd April 2, 2017
+.Dt GBZ80 7
+.Os RGBDS Manual
+.Sh NAME
+.Nm rgbds
+.Nd gbz80 opcode reference
+.Sh DESCRIPTION
+This is the list of opcodes supported by
+.Nm ,
+including a short description, the number of bytes needed to encode them and
+the number of CPU cycles at 1MHz
+.Pq or 2MHz in GBC dual speed mode
+needed to complete them.
+.Pp
+.Sh LEGEND
+List of abbreviations used in this document.
+.Bl -tag
+.It Sy r8
```

Comment:
`Sy` is appropriate for A, B, C, D, E, F, H, L, AF, BC, DE, HL, etc. But `Ar` should be used for r8, n8, r16, n16, and e8, since they’re arguments, not literal symbols.